src/Clients.php

```diff
@@ -143,6 +169,14 @@ public function person($data) {
 
 	}
 
+	public function id($id) {
```

Comment:
might be better to call this `setIt()` so its more clear for others.

Reply:
I have the remaining nomenclature in there referring to each method as a property, no true getters or setters. I'd be open to it, but I think it would break the pattern. Would`nestioClientId` be better?

Reply:
nah its fine if there is a set pattern, would rather just get this out and we can revisit it in the future :)

Reply:
💯